Sources/MapboxMaps/Foundation/Camera/CameraTransition.swift

```diff
@@ -0,0 +1,113 @@
+import UIKit
+import CoreLocation
+
+/// Structure used to represent a desired change to the map's camera
+public struct CameraTransition {
+
+    /// Represents a change to the center coordinate of the map.
+    /// NOTE: Setting the `toValue` of `center` overrides any `anchor` animations
+    public var center: Change<CLLocationCoordinate2D>
+
+    /// Represents a change to the zoom of the map.
+    public var zoom: Change<CGFloat>
+
+    /// Represetns a change to the padding of the map.
+    public var padding: Change<UIEdgeInsets>
+
+    /// Represents a change to the anchor of the map
+    /// NOTE: Incompatible with concurrent center animations
+    public var anchor: Change<CGPoint>
+
+    /// Set this value in order to make bearing/zoom animations
+    /// honor a constant anchor throughout the transition
+    /// NOTE: Incompatible with concurrent center animations
+    public var constantAnchor: CGPoint?
```

Comment:
Not sure I understand the need for this

Reply:
and then you can clean up the update logic

Reply:
This is done.